pkg/proxy/proxyinfo.go

```diff
@@ -58,15 +59,20 @@ func GetProxyInfo(kubeClient kube.CLIClient, istioNamespace string) (*[]istioVer
 	return &pi, nil
 }
 
-// GetIDsFromProxyInfo is a helper function to retrieve list of IDs from Proxy.
-func GetIDsFromProxyInfo(kubeClient kube.CLIClient, istioNamespace string) ([]string, error) {
+// GetUserLevelIDsFromProxyInfo is a helper function to retrieve list of IDs from Proxy.
+// This list only retrieves proxies at the user level, while proxies at the system level (such as gateway and ztunnel) will be ignored.
+func GetUserLevelIDsFromProxyInfo(kubeClient kube.CLIClient, istioNamespace string) ([]string, error) {
```

Comment:
The name Userlevel seems a little bit wired to me, as I don't really know what is user level proxies. I think we should rename this to a more proper one.

Reply:
good idea